static/app/components/core/code/inlineCode.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled';
+
+export const InlineCode = styled('code')`
+  /**
+   * Reset any properties that might be set by the global CSS styles.
+   */
+  margin: 0;
+  padding: 0;
+  border: none;
+
+  font-family: ${p => p.theme.text.familyMono};
+  /**
+   * adjust height of x character to 57% of bounding box
+   * to match Rubik (see )
+   */
+  font-size-adjust: ex-height 0.57;
+
+  color: ${p => p.theme.tokens.content.promotion};
+  background: color-mix(in srgb, currentColor, transparent 93%);
```

Comment:
would this help with the issues we have with the code snippets (https://linear.app/getsentry/issue/DE-277/dark-mode-code-snippet-background-matches-main-background) ?